requirements.txt

```diff
@@ -1,3 +1,3 @@
-PyLint==1.6.5
-pep8>=1.6.0,<=1.6.2
-modernize==0.5
+PyLint>=1.7.1
+pycodestyle>=2.3.1
+modernize>=0.5
```

Comment:
We can't just set lower bounds on requirements because different versions result in different lint outcomes, that's why it was hard coded.
I don't mind bumping versions but this has to be coordinated with the Salt project or we'll have new lint failures going through the roof.